application/src/main/resources/locale/blueprint/horizons/description.csv

```diff
@@ -1,114 +1,114 @@
-blueprint.description.horizons.engineer_b1_invite,Gain access: Gain exploration rank Surveyor or higher.,Gain access: Gain exploration rank Surveyor or higher.,Zugriff auf Gewinn: Erkundungsrangvermesser oder höher.,Obtenga acceso: Gane Surveyor de rango de exploración o superior.,Accédez à: Gain Exploration Rank-Surveyor ou supérieur.,Ganhe acesso: ganho de exploração Surveyor ou superior.,Доступ к получению: экспертиз ранга разведки или выше.
-blueprint.description.horizons.engineer_b1_learn,Learn about: Reach level 3 and 33% reputation with The Dweller.,Learn about: Reach level 3 and 33% reputation with The Dweller.,Erfahren Sie mehr über: Ruf des Level 3 und 33% mit dem Bewohner.,Aprenda sobre: alcanzar la reputación de nivel 3 y 33% con el habitante.,Renseignez-vous sur: Atteindre la réputation de niveau 3 et 33% avec le habitant.,Aprenda sobre: Alcance Nível 3 e 33% de reputação com o morador.,Узнайте о репутации уровня 3 и 33% с жителем.
-blueprint.description.horizons.engineer_b1_unlock,Initial contract: Mine 10T of Osmium.,Initial contract: Mine 10T of Osmium.,Erstvertrag: Mine 10t Osmium.,Contrato inicial: Mine 10t de Osmium.,Contrat initial: mien 10t d''osmium.,Contrato inicial: mina 10t de ósmio.,Первоначальный контракт: шахта 10T осмия.
-blueprint.description.horizons.engineer_b2_invite,Gain access: When you have bought or sold goods at fifty different stations.,Gain access: When you have bought or sold goods at fifty different stations.,Zugang erhalten: Wenn Sie Waren an fünfzig verschiedenen Stationen gekauft oder verkauft haben.,Obtenga acceso: cuando haya comprado o vendido productos en cincuenta estaciones diferentes.,Accédez: lorsque vous avez acheté ou vendu des marchandises dans cinquante stations différentes.,Ganhe acesso: quando você comprar ou vender mercadorias em cinquenta estações diferentes.,Получите доступ: когда вы купили или продали товары на пятьдесят разных станциях.
-blueprint.description.horizons.engineer_b2_learn,Learn about: Reach level 3 and 33% reputation with The Dweller.,Learn about: Reach level 3 and 33% reputation with The Dweller.,Erfahren Sie mehr über: Ruf des Level 3 und 33% mit dem Bewohner.,Aprenda sobre: alcanzar la reputación de nivel 3 y 33% con el habitante.,Renseignez-vous sur: Atteindre la réputation de niveau 3 et 33% avec le habitant.,Aprenda sobre: Alcance Nível 3 e 33% de reputação com o morador.,Узнайте о репутации уровня 3 и 33% с жителем.
+blueprint.description.horizons.engineer_b1_invite,Gain access: Gain exploration rank Surveyor or higher.,Gain access: Gain exploration rank Surveyor or higher.,Zugriff auf Gewinn: Erkundungsrangvermesser oder höher.,Obtenga acceso: Gane Surveyor de rango de exploración o superior.,Invitation: Gagner le rang d’’éclaireur ou supérieur en exploration,Ganhe acesso: ganho de exploração Surveyor ou superior.,Доступ к получению: экспертиз ранга разведки или выше.
```

Comment:
alright, I dove a bit deeper in this now. Looks like 2 `’ RIGHT SINGLE QUOTATION MARK(UTF+2019)` already exist in the localization files. It seems that those do not have to be escaped like the `' APOSTROPHE(UTF+0027)`. Even though the APOSTROPHE is the default on my keyboard, the RIGHT SINGLE QUOTATION MARK looks better since we don't have to escape it. I'll just have to safeguard it so the APOSTROPHE isn't used anymore. If you could change the double RIGHT SINGLE QUOTATION MARK to a single one in this file, them I will take care of conversion of the APOSTROPHE later. I'll put a task where it needs to be done.